propono.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 2.1"
   spec.add_development_dependency "rake"
   spec.add_development_dependency "mocha"
+  spec.add_development_dependency "nokogiri"
```

Comment:
Still not clear on why we need this?

Reply:
In ruby 3.x, the AWS library needs to have an XML parser. I don't know why it doesn't require this in 2.x. What I do know is that if you leave this line out and test under 3.x, you'll get the following error:

```
/Users/adavis/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/aws-sdk-core-3.125.4/lib/aws-sdk-core/xml/parser.rb:74:in `set_default_engine': Unable to find a compatible xml library. Ensure that you have installed or added to your Gemfile one of ox, oga, libxml, nokogiri or rexml (RuntimeError)
```

I picked nokogiri because it's a known quantity that I have installed everywhere, we can pick anything. I left it as a dev dependency so the project that includes propono will resolve the dependency (because it likely already has).